src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/TaxonTranslation.xml

```diff
@@ -7,14 +7,21 @@
     <resource class="%sylius.model.taxon_translation.class%" shortName="TaxonTranslation">
         <attribute name="validation_groups">sylius</attribute>
 
+        <attribute name="normalization_context">
+            <attribute name="groups">
+                <attribute>admin:taxon_translation:read</attribute>
+                <attribute>admin:taxon_translation:read</attribute>
```

Comment:
Duplicate? I think you wanted to add shop as second. And same question as with ProductVariant/Taxon